Makefile

```diff
@@ -947,6 +947,7 @@ upgrade-mlrun-kfp-deps-lock: verify-uv-version ## Upgrade mlrun-kfp locked requi
 	uv pip compile \
 		requirements.txt \
 		dockerfiles/mlrun-kfp/requirements.txt \
+		--python-version 3.9 \
```

Comment:
Why is that needed?

Reply:
mlrun/mlrun-kfp runs only with python 3.9